index.html

```diff
@@ -2,11 +2,14 @@
   <html lang="en">
   <head>
      <meta charset="UTF-8">
-     <title>Angular ToDo App</title>
+     <title>Brittany's Daily To Do List</title>
      <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.2/angular.min.js"></script>
      <script src="app.js"></script>
   </head>
-  <body>
-    <h1>To Do!</h1>
+  <body data-ng-controller="toDosController" as vm>
```

Comment:
The directive `data-ng-controller` needs to have " " around the entire statement, so instead of `"toDosController"` as vm it should be `"todosController as vm"`. Also, since you called it `"todosController"` in your `app.js` the name must be spelled and consistent here